features/src/rapids-build-utils/opt/rapids-build-utils/manifest.yaml

```
@@ -6,7 +6,7 @@ x-git-defaults: &git_defaults
 x-rapids-build-backend-args: &rapids_build_backend_args |
   --config-settings skbuild.strict-config=false
   --config-settings rapidsai.disable-cuda=true
-  --config-settings rapidsai.matrix_entry=cuda=$(grep -o '^[0-9]+.[0-9]+' <<< "${CUDA_VERSION}")
+  --config-settings rapidsai.matrix-entry=cuda=$(grep -Po '^[0-9]+.[0-9]+' <<< "${CUDA_VERSION}")
```

Why the addition of the `-P` flag? Also, shouldn't it be:

```suggestion
  --config-settings rapidsai.matrix-entry=cuda=$(grep -Po '^[0-9]+\.[0-9]+' <<< "${CUDA_VERSION}")
```

(notice the `\` before the `.`)?

---

Doesn't seem to make a difference:
```
$ grep -o '^[0-9]+.[0-9]+' <<< 12.5.1.123 || echo $?
1
$ grep -Po '^[0-9]+.[0-9]+' <<< 12.5.1.123 || echo $?
12.5
$ grep -Po '^[0-9]+\.[0-9]+' <<< 12.5.1.123 || echo $?
12.5
```